CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## [Unreleased]
+## 1.0.4 (17.06.2019)
```

Comment:
Please don't remove ## [Unreleased] section. It will be for further use.

Reply:
done